group_vars/artemis_prod_like_common.yml

```
@@ -96,3 +96,11 @@
 artemis_deployment_user_uid: 1338
 artemis_deployment_user_public_key: "{{ lookup('hashi_vault', 'kv/data/artemis/' + var_vault_group).get('ssh_pub') }}"
 artemis_deployment_user_comment: "User to deploy artemis to this host"
+
+
+##############################################################################
+# Other Configuration
+##############################################################################
+
+# Lower the entropy bits to fix a bug with TSAN and MSAN C exercises
+entropy_bits_override: 30
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)